src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -39,17 +43,20 @@
     private final ServiceConfigProvider serviceConfigProvider;
     private final TransformationClient transformationClient;
     private final AuthTokenGenerator s2sTokenGenerator;
+    private final CoreCaseDataApi feignCcdApi;
 
     public CreateCaseCallbackService(
         CreateCaseValidator validator,
         ServiceConfigProvider serviceConfigProvider,
         TransformationClient transformationClient,
-        AuthTokenGenerator s2sTokenGenerator
+        AuthTokenGenerator s2sTokenGenerator,
+        CoreCaseDataApi feignCcdApi
```

Comment:
How about `ccdApi`?

Reply:
Mixes up with our own CcdApi in codebase. This reeks for general review and unification. tl;dr - could not re-use CcdApi. But perhaps for this class purposes - can be renamed, np